frontend/src/functions/getPlan.ts

```diff
@@ -45,8 +59,16 @@ export const getPhoneSubscribeLink = (
   return `${runtimeData.FXA_ORIGIN}/subscriptions/products/${runtimeData.PHONE_PRODUCT_ID}?plan=${plan.id}`;
 };
 
-export const getBundlePrice = (runtimeData: RuntimeDataWithBundleAvailable) => {
-  return getPlan(runtimeData.BUNDLE_PLANS, "yearly").price;
+export const getBundlePrice = (
+  runtimeData: RuntimeDataWithBundleAvailable,
+  l10n: ReactLocalization
```

Comment:
question (non-blocking): For learning on my end, why do we have to pass `l10n` in every time we call this function - we can't call it from `getPlan`?

Reply:
Detecting the user's language happens at runtime, and the correct localisation strings are then passed into the React app [via a Context Provider](https://github.com/mozilla/fx-private-relay/blob/adf9164f5f19e4dd5dfc65d2ec9f26a22ad48569/frontend/src/pages/_app.page.tsx#L84). React components within that provider can then access it via the `useLocalization` hook, but outside of components (e.g. in the `getPlan` function), there's no way to get to that. (Or at least, I don't think there is - it might be possible to call `getL10n` outside of the Provider and get access to the same instance, but even if it is possible, I like to keep functions [pure](https://en.wikipedia.org/wiki/Pure_function) if possible, to make it easier to grasp their scope and make them easier to test if needed.)